asset/scss/style.scss

```
@@ -0,0 +1,381 @@
+    }
+  }
+  .btn-left {
+    color: $white-color;
+  }
+  .btn-right {
+    border-color: $primary-color;
+    color: $primary-color;
+  }
+}
+
+.introduce__pagination {
+  padding: 148px 0 46px;
+}
+.dot {
+  height: 20px;
+  width: 20px;
+  border: 4px solid $bg-color;
+  background-color: transparent;
+  border-radius: 50%;
+  display: inline-block;
+  margin: 0 8px;
+  outline: 3px solid $white-color;
+}
+.dot.dot-active {
+  background-color: $primary-color;
+}
+
+// Services
+.services {
+  padding: 142px 0;
+  color: $secondary-color;
+}
+.service-wrap {
+  width: 1170px;
+  margin: 0 auto;
+}
+
+.services-title {
+  font-size: 3.6rem;
+  text-align: center;
+}
+.services-description {
+  line-height: 1.5;
+  font-size: 1.6rem;
+  margin: 80px 0;
+}
+
+.services-products {
+  display: flex;
+}
+
+.products-items {
+  flex-basis: 25%;
+  text-align: center;
+  border: 1px solid #dfe5e8;
+}
+.product-title {
+  font-size: 1.8rem;
+  padding: 36px 0;
+  background-color: $color-1;
+}
+
+.product-price {
+  background-color: $color-2;
+  color: $white-color;
+  padding: 36px 0;
+
+  h4 {
+    font-size: 5rem;
+    font-weight: 200;
+  }
+  p {
+    font-size: 1.4rem;
+    margin-top: 12px;
+  }
+}
+
+.product-description {
+  background-color: $color-1;
+  li {
+    font-size: 1.4rem;
+    padding: 20px 0;
+    border-top: 1px solid #dfe5e8;
+  }
+
+  button {
+    margin: 24px 0 50px;
+    font-size: 1.2rem;
+    font-weight: 600;
+    padding: 16px 24px;
+    border: none;
+    border-radius: 30px;
+    background-color: $btn-color;
+    color: $white-color;
+    cursor: pointer;
+    transition: all 0.3s;
+
+    &:hover {
+      background-color: $color-4;
+    }
+
+    i {
+      margin-right: 6px;
+    }
+  }
+}
+
+.product-sub-description {
+  text-align: center;
+  height: 80px;
+
+  h4 {
+    font-size: 1.8rem;
+    font-weight: normal;
+    padding: 20px 0;
+  }
+  p {
+    font-size: 1.2rem;
+    opacity: 0.8;
+  }
+}
+
+.products-items:nth-child(2) {
+  .product-price {
+    background-color: $color-3;
+  }
+}
+.products-items:nth-child(3) {
+  position: relative;
+
+  button {
+    background-color: $color-4;
+  }
+
+  .product-price {
+    background-color: $color-4;
+  }
+
+  .product-title {
+    background-color: $white-color;
+    h3 {
+      font-size: 1.8rem;
+      font-weight: normal;
+    }
+    p {
+      font-size: 1.4rem;
+      padding-top: 16px;
+    }
+  }
+}
+
+.product-new-tag {
+  position: absolute;
+  padding: 20px 10px;
+  border-radius: 50%;
+  background-color: $bg-color;
+  color: $white-color;
+  top: 0;
+  left: 20%;
+  transform: translate(-50%, -50%);
+  font-size: 1.2rem;
+  font-weight: 600;
+}
+
+.products-items:nth-child(4) {
+  .product-price {
+    background-color: $color-5;
+  }
+}
+
+.products-items:nth-child(4),
+.products-items:nth-child(2),
+.products-items:nth-child(1) {
+  margin-top: 30px;
+}
+
+.product-description {
+  li:nth-of-type(3),
+  li:nth-of-type(4),
+  li:nth-of-type(5) {
```

tương tự em đổi cách viết selector để dễ maintain và tái sử dụng nhé, tham khảo cách viết của bootstrap 
```suggestion
.text-decoration-line-through {
  text-decoration: line-through;
}
```